types/src/ledger_info.rs

```diff
@@ -85,6 +85,11 @@ impl LedgerInfo {
         &self.commit_info
     }
 
+    /// The epoch after the block committed
+    pub fn post_epoch(&self) -> u64 {
```

Comment:
Not sure naming is intuitive (maybe due to my poor English), `epoch_next_block` is more fool friendly but I'm not sure if that's elegant.

Reply:
how about `next_epoch`

Reply:
that's confusing.. epoch of the next block can be the same or +1, but next epoch is always +1, right?

Reply:
fine, english is hard